modules/@apostrophecms/ui/ui/apos/scss/mixins/_theme_mixins.scss

```diff
@@ -49,3 +22,90 @@ $input-color-disabled: var(--a-base-4);
   --a-primary-button-disabled: #{mix(white, $color, 40%)};
   // stylelint-enable
 }
+
+//Application Styles
+
+@mixin typeSmall {
+  font-family: var(--a-family-default);
+  font-size: var(--a-type-small);
+  font-weight: var(--a-weight-base);
+  letter-spacing: var(--a-letter-base);
+  line-height: var(--a-line-tall);
+}
+
+@mixin typeBase {
+  font-family: var(--a-family-default);
+  font-size: var(--a-type-base);
+  font-weight: var(--a-weight-base);
+  letter-spacing: var(--a-letter-base);
+  line-height: var(--a-line-base);
+}
+
+@mixin typeTitle {
+  margin: 0;
+  font-family: var(--a-family-default);
+  font-size: var(--a-type-heading);
+  font-weight: var(--a-weight-base);
+  letter-spacing: var(--a-letter-base);
+  line-height: var(--a-line-tall);
+}
+
+@mixin typeDisplay {
+  margin: 0;
+  font-family: var(--a-family-default);
+  font-size: var(--a-type-display);
+  font-weight: var(--a-weight-base);
+  letter-spacing: var(--a-letter-base);
+  line-height: var(--a-line-base);
+}
+
+@mixin typeAffordance {
```

Comment:
kebab .. Also this name is vague when seeing it in code .. is there something more concrete

Reply:
kebabed, and renamed "type help"